catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java

```diff
@@ -639,7 +628,8 @@ private ResourceInfo getResourceInfo(
 
   private void validateResourceInfoRequest(String name, Object value)
       throws ResourceNotSupportedException {
-    if (!(ResourceRequest.GET_RESOURCE_BY_PRODUCT_URI.equals(name) && value instanceof URI)
+    if (!(ResourceRequest.GET_RESOURCE_BY_PRODUCT_URI.equals(name)
+            || name.equals(Core.DERIVED_RESOURCE_URI) && value instanceof URI)
```

Comment:
Just noticed the logic isn't what I intended here. I'll fix it on my next run through.